packages/cli/src/ceramic-daemon.ts

```diff
@@ -778,6 +803,8 @@ export class CeramicDaemon {
     }
 
     const indexedModels = await this.ceramic.admin.getIndexedModels()
+    console.log(`RECORDING NUMBER INDEXED MODELS: {indexedModels.length}`)
+    ModelMetrics.observe(Observable.TOTAL_INDEXED_MODELS, indexedModels.length)
```

Comment:
How often are we collecting these metrics? I don't remember which relation type you said the model uses. Is it SINGLE or LIST? 

Also who is the controller for the stream? The did created from the node seed?

Reply:
yeah its the same DID used to authenticate with CAS, set here https://github.com/ceramicnetwork/js-ceramic/blob/00ceb58c9d9e785f07d9d4349d464a4f5b32199c/packages/cli/src/ceramic-daemon.ts#L325

(we dont' set this explicitly we get it from the ceramic daemon acting as an instance of CeramicAPI)

I did not set it explicitly, maybe should have, it would be LIST - there can be many for the DID.
https://github.com/ceramicnetwork/observability/blob/master/packages/model-metrics/src/composites/00-simpleNodeMetrics.graphql#L25

@nathanielc please feel free to request changes in observability package and I can make - i did not review all the PRs there as i was creating it from scratch